calico/networking/ip-autodetection.md

```diff
@@ -50,39 +50,105 @@ By default, {{site.prodname}} uses the **first-found** method; the first valid I
 
-Because you can configure IP address and subnet using either environment variables or node resource, the following table describes how values are synchronized.
+- **IP or domain name**
 
-| **If this environment variable...** | **Is...**                                             | **Then...**                                                  |
-| ----------------------------------- | ----------------------------------------------------- | ------------------------------------------------------------ |
-| IP/IP6                              | Explicitly set                                        | The specified values are used, and the Node resource is updated. |
-|                                     | Set to autodetect                                     | The requested method is used (first-found, can-reach, interface, skip-interface, kubernetes-internal-ip), and the Node resource is updated. |
-|                                     | Not set, but Node resource has IP/IP6 values          | Node resource value is used.                                 |
-| IP                                  | Not set, and there is no IP value in Node resource    | Autodetects an IPv4 address and subnet, and updates Node resource. |
-| IP6                                 | Not set, and there is a no IP6 value in Node resource | No IP6 routing is performed on the node.                     |
+  A reachable destination (IP address or domain). For example:
 
-### How to
+  ```
+  kind: Installation
+  apiVersion: operator.tigera.io/v1
+  metadata:
+    name: default
+  spec:
+    calicoNetwork:
+      nodeAddressAutodetectionV4:
+        canReach: 8.8.8.8
+  ```
 
-- [Change the autodetection method](#change-the-autodetection-method)
-- [Manually configure IP address and subnet for a node](#manually-configure-ip-address-and-subnet-for-a-node)
+- **Including matching interfaces**
 
-#### Change the autodetection method
+  A regular expression in Golang syntax that includes interfaces that match. For example:
+
+  ```
+  kind: Installation
+  apiVersion: operator.tigera.io/v1
+  metadata:
+    name: default
+  spec:
+    calicoNetwork:
+      nodeAddressAutodetectionV4:
+        interface: eth.*
+  ```
+
+- **Excluding matching interfaces**
+
+  A regular expression in Golang syntax that excludes interfaces that match. For example:
+
+  ```
+  kind: Installation
+  apiVersion: operator.tigera.io/v1
+  metadata:
+    name: default
+  spec:
+    calicoNetwork:
+      nodeAddressAutodetectionV4:
+        skipInterface: eth.*
+  ```
+
+- **Including CIDRs**
+
+  A list of IP ranges in CIDR format to determine valid IP addresses on the node to choose from. For example:
+
+  ```
+  kind: Installation
+  apiVersion: operator.tigera.io/v1
+  metadata:
+    name: default
+  spec:
+    calicoNetwork:
+      nodeAddressAutodetectionV4:
+        cidrs:
+          - "192.168.200.0/24"
+  ```
 
+- **Kubernetes Node IP**
+
+  An internal IP address assigned to the Kubernetes node.
```

Comment:
I think we should explicitly call out that this comes from the `Node.Status.Addresses` field

Reply:
And maybe move this one to the top - I think this is probably the most often desired auto-detection method.

Reply:
I put that detail in the node configuration reference page but wasn't sure if it should go in this page or not. I think you're right though